content/docs/connectors/keystone.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Authentication Through OpenStack Keystone"
+linkTitle: "OpenStack Keystone"
+description: ""
+date: 2020-09-30
+draft: false
+toc: true
+weight: 2100
```

Comment:
This has the same weight as Bitbucket Cloud, so they'll default to alphabetical ordering between them.

If we want Keystone between Bitbucket cloud and Openshift, I'd suggest a weight between the two:

```suggestion
weight: 2105
```

Reply:
I think the right place is after Gitea connector, so `2135`.